lib/ip-locator.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const Ipinfo = require('ipinfo')
+const RequestIp = require('request-ip')
+
+class IpLocator {
+  constructor (options) {
+    const { enabledByDefault = true } = options
+    this.enabledByDefault = enabledByDefault
+  }
+
+  /**
+   * @function setting
+   * @description Return the setting for the hapi-geo-locate by taking the request as a parameter and return the setting
+   * @param {Object} request
+   * @returns {Object}
+   */
+  setting (request) {
```

Comment:
please call this method `routeConfig`

```suggestion
  routeConfig (request) {
```